declarations/WebpackOptions.d.ts

```diff
@@ -38,7 +38,36 @@ export type Dependencies = string[];
 /**
  * A developer tool to enhance debugging (false | eval | [inline-|hidden-|eval-][nosources-][cheap-[module-]]source-map).
  */
-export type DevTool = (false | "eval") | string;
+export type DevTool =
+	| (
+			| false
+			| "eval"
+			| "eval-cheap-source-map"
+			| "eval-cheap-module-source-map"
+			| "eval-nosources-cheap-source-map"
+			| "eval-nosources-cheap-module-source-map"
+			| "eval-nosources-source-map"
+			| "eval-source-map"
+			| "cheap-source-map"
+			| "cheap-module-source-map"
+			| "source-map"
+			| "inline-cheap-source-map"
+			| "inline-cheap-module-source-map"
+			| "inline-source-map"
+			| "inline-nosources-cheap-source-map"
+			| "inline-nosources-cheap-module-source-map"
+			| "inline-nosources-source-map"
+			| "nosources-cheap-source-map"
+			| "nosources-cheap-module-source-map"
+			| "nosources-source-map"
+			| "hidden-nosources-cheap-source-map"
+			| "hidden-nosources-cheap-module-source-map"
+			| "hidden-nosources-source-map"
+			| "hidden-cheap-source-map"
+			| "hidden-cheap-module-source-map"
+			| "hidden-source-map"
+	  )
+	| string;
```

Comment:
I wonder if this way code-compilation works or if this is normalized to `false | string` by typescript and code-compilation doesn't work...

Reply:
looks like you are correct about `| string`, take a look [repl](https://www.typescriptlang.org/play?#code/C4TwDgpgBAglC8UDkBDJUA+yBG6sDMUAbAZwgChRIoAhBZNTHPKE4AJwEsA7AcycKkK5AMYB7bmygoAXLHqokoiVJSyG6RCnLLJwKNjl1Ei6SSji9uqdkMb62ckA)